test/integration/test_ibm_job.py

```diff
@@ -0,0 +1,438 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""IBMJob Test."""
+import copy
+import time
+from datetime import datetime, timedelta
+from threading import Thread, Event
+from unittest import SkipTest, mock
+from unittest import skip
+
+from dateutil import tz
+from qiskit import ClassicalRegister, QuantumCircuit, QuantumRegister
+from qiskit.compiler import transpile
+from qiskit.providers.jobstatus import JobStatus, JOB_FINAL_STATES
+from qiskit.test.reference_circuits import ReferenceCircuits
+
+from qiskit_ibm_provider.api.rest.job import Job as RestJob
+from qiskit_ibm_provider.exceptions import IBMBackendApiError
+
+from qiskit_ibm_runtime import IBMBackend, RuntimeJob
+from qiskit_ibm_runtime.api.exceptions import RequestsApiError
+from qiskit_ibm_runtime.exceptions import RuntimeJobTimeoutError, RuntimeJobNotFound
+from ..decorators import (
+    IntegrationTestDependencies,
+    integration_test_setup_with_backend,
+)
+from ..fake_account_client import BaseFakeAccountClient, CancelableFakeJob
+from ..ibm_test_case import IBMIntegrationTestCase
+from ..utils import (
+    most_busy_backend,
+    cancel_job_safe,
+    submit_and_cancel,
+)
+
+
+class TestIBMJob(IBMIntegrationTestCase):
+    """Test ibm_job module."""
+
+    sim_backend: IBMBackend
+    real_device_backend: IBMBackend
+    bell = QuantumCircuit
+    sim_job: RuntimeJob
+    last_month: datetime
+
+    @classmethod
+    @integration_test_setup_with_backend(simulator=False, min_num_qubits=2)
+    def setUpClass(cls, backend: IBMBackend, dependencies: IntegrationTestDependencies) -> None:
+        """Initial class level setup."""
+        # pylint: disable=arguments-differ
+        super().setUpClass(dependencies=dependencies)
+        cls.sim_backend = dependencies.service.backend(
+            "ibmq_qasm_simulator", instance=dependencies.instance
```

Comment:
Same thing here, we should remove the instance parameter here bc it won't work with the cloud channel

Reply:
Done.